app/src/main/java/org/mozilla/fenix/composables/ListItemTabLarge.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.composables
```

Comment:
Can we refactor the naming of this package to be just `compose` instead of `composables` since we already use that naming within AC.

Reply:
Sure. Thanks!